tests/lib/rules-preprocessor/gjs-gts-processor-test.js

```diff
@@ -79,16 +79,17 @@ const valid = [
       </template>
     `,
   },
-  {
-    filename: 'my-component.gjs',
-    code: `
-      const Foo = <template>hi</template>
-
-      <template>
-        <Foo />
-      </template>
-    `,
-  },
+// SKIP this scenario. Tracked in https://github.com/ember-cli/eslint-plugin-ember/issues/1685
+//   {
```

Comment:
Let's use a `/* */` multi-line comment to avoid touching the git blame history for all these lines.